.github/workflows/release.yml

```
@@ -1,13 +1,33 @@
-        description: 'Version to release (e.g., v0.1.0)'
+      scope:
+        description: 'Comma-separated package scope (optional)'
         required: false
-        type: string
-
-# ... rest of the job unchanged ...
\ No newline at end of file
+        default: ''
+jobs:
+  lint:
+    uses: ./.github/workflows/lint.yml  # reuse if using reusable workflows; else inline
```

actionlint (1.7.8)

error while parsing reusable workflow "./.github/workflows/lint.yml": "workflow_call" event trigger is not found in "on:" at line:3, column:3

(workflow-call)